packages/gasket-nextjs/test/layout/with-gasket-data.test.js

```diff
@@ -0,0 +1,72 @@
+import { jest, expect } from '@jest/globals';
+import { createElement, Children } from 'react';
+
+jest.unstable_mockModule('../../lib/server/request.js', () => ({
+  request: jest.fn().mockReturnValue({})
+}));
+
+describe('withGasketData', () => {
+  let mockGasket, mockPublicGasketData, mockLayout, layout;
+
+  beforeAll(async () => {
+    layout = (await import('../../lib/layout/with-gasket-data.js'));
+  });
+
+  beforeEach(async () => {
+    mockPublicGasketData = { foo: 'bar' };
+    mockGasket = {
+      actions: {
+        getPublicGasketData: jest.fn(() => mockPublicGasketData)
+      }
+    };
+
+    const Html = (props) => createElement('html', { ...props });
+    const Main = (props) => createElement('main', { ...props });
+    const Head = (props) => createElement('head', { ...props });
+    const NextScript = (props) => createElement('script', { 'data-testid': 'next-script', ...props });
+
+    /**
+     *
+     */
```

Comment:
Either give it a jsdoc or disable since it's a test helper
```suggestion
// eslint-disable-next-line jsdoc/require-jsdoc
```